populate_course_run_reporting_type_and_announcement.py

```diff
@@ -0,0 +1,80 @@
+import argparse
```

Comment:
Try a relative import: `from ..services`. Otherwise, we need to bite the bullet and fix this repo.